NuGet.Config

```diff
@@ -4,6 +4,7 @@
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="Local" value="tools/LocalNugetFeed" />
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
+    <add key="azure-sdk-for-net-dev-ops" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
     <add key="azure-sdk-for-net" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-net/index.json" />
```

Comment:
For our usage here please go ahead and remove the blob feed.

Reply:
Well you might need to wait until we get more packages published to our devops feed before you can remove this but it might be worth a test to see if we are depending on any of those currently. I bet the smoke tests are depending on it at least.

Reply:
@chidozieononiwu I think that we have now had enough time for packages to be published.  We should remove the blob feed and update documentation on our dev feed. Once that is done (give it a week or so) then we stop publishing to our blob feed.